src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -444,7 +439,7 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                         </div>
 
                         {
-                            (!this.resultsViewPageStore.genesInvalid) && (
+                            (!this.resultsViewPageStore.genesInvalid && !this.resultsViewPageStore.isQueryInvalid) && (
```

Comment:
like for example, genesInvalid should probably be geneNamesInvalid and should be part of isQueryValid.  but we should follow up with that.  we don't need to handle this now.